app/create_file.py

```diff
@@ -1 +1,43 @@
-# write your code here
+import sys
+import os
+import datetime
+from typing import List
+
+
+def create_path(directories: List[str]) -> None:
+    os.makedirs(os.path.join(*directories), exist_ok=True)
+
+
+def add_content_to_file(file_path: str) -> None:
+    with open(os.path.join(*file_path), "a") as file:
+        file.write(
+            datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+            + "\n"
+        )
+        while True:
+            content_line = input("Enter content line:")
```

Comment:
Add a space after `:`
```suggestion
            content_line = input("Enter content line: ")
```